pom.xml

```diff
@@ -133,7 +158,7 @@
 				<plugin>
 					<groupId>org.jooq</groupId>
 					<artifactId>jooq-codegen-maven</artifactId>
-					<version>3.19.13</version>
+					<version>3.19.11</version>
```

Comment:
What was the reason to change the JOOQ-version?

Reply:
I needed to change it, my IDE was giving me a mismatch warning with the version number.